packages/sanes-browser-extension/config/staging.json

```diff
@@ -30,5 +30,6 @@
       }
     }
   ],
+  "walletCreationDisabled": true,
```

Comment:
Should be false?

Reply:
No, in staging (i.e. testnets), the new address generator should be used as well (see https://github.com/iov-one/address-generator). This is to make testers familiar with the tooling.